### ISSUE TITLE
Test out of order

### ISSUE THREAD

**Issue by jhuels** [hbenl/vscode-test-explorer#55]
I believe the last VS Code update broke the ordering of tests.  If I recall correctly, they used to be in alphabetical order.

VS Code: 1.32.1

![image](https://user-images.githubusercontent.com/40254454/54934088-265dda00-4edb-11e9-95d8-00e4b823144f.png)

**Comment by matepek** [comment]
What is the name of your extension? (Not the Test Explorer)

**Comment by jhuels** [comment]
Catch2 and Google Test Explorer. 
Sorry I'm not familiar with their relationship.  Catch2 provides the interface for populating, and test-explorer is simply the UI to display?

**Comment by jhuels** [comment]
Sorry, I was missing, `testExplorer.sort`.

**Comment by matepek** [comment]
Yes, exactly.
I am the creator of the _Catch2 and Google Test Explorer_.
I did the sorting until the _Test Explorer_ has implemented they own sorting mechanisms.
So I deleted my sorting mechanism, but the user (you) has to set it manually.
With this `testExplorer.sort` = `byLable` you will get back the old sorting. If you like it like this, set it for your [user settings, not in the workspace settings](https://code.visualstudio.com/docs/getstarted/settings).
See [here](https://github.com/hbenl/vscode-test-explorer#configuration) for more options.
